uw_sws/section.py

```diff
@@ -190,8 +192,34 @@ def _get_sections_by_person_and_term(person,
         params.append(("delete_flag", ','.join(sorted(delete_flag)),))
 
     url = "%s?%s" % (section_res_url_prefix, urlencode(params))
+    return get_resource(url)
 
-    return _json_to_sectionref(get_resource(url))
+
+def get_last_section_by_instructor_and_terms(person,
+                                             term,
+                                             future_terms,
+                                             transcriptable_course='all',
+                                             delete_flag=['active']):
+    try:
+        raw_resp = _get_sections_by_person_and_term(person,
+                                                    term,
+                                                    "Instructor",
+                                                    False,
+                                                    future_terms,
+                                                    transcriptable_course,
+                                                    delete_flag)
+    except DataFailureException as ex:
+        if ex.status == 404:
+            return None
+        raise
```

Comment:
Should this be:
raise ex

rather than:
raise
?

Reply:
That is what it's doing, calling raise with no args raises the last exception.